common/lib/common-definitions/src/logger.ts

```diff
@@ -41,10 +41,11 @@ export interface ITelemetryBaseEvent extends ITelemetryProperties {
  */
 export interface ITelemetryBaseLogger {
     /**
-     * An optional boolean which indicates to the user of this interface that tags (i.e. `ITaggedTelemetryPropertyType`
-     * objects) are in use. Eventually this will be a required property, but this is a stopgap that allows older hosts
-     * to continue to pass through telemetry without trouble (this property will simply show up undefined), while our
-     * current logger implementation in `telmetry-utils` handles tags in a separate manner.
+     * @deprecated An optional boolean which indicates to the user of this interface that tags
+     * (i.e. `ITaggedTelemetryPropertyType` objects) are in use. Eventually this will be a required property, but this
+     * is a stopgap that allows older hosts to continue to pass through telemetry without trouble (this property will
+     * simply show up undefined), while our current logger implementation in `telmetry-utils` handles tags in a
+     * separate manner.
```

Comment:
I'd delete the whole comment and just leave the @deprecated tag

Reply:
For tsdoc rules, can't just have the tag by itself so had to go back and add a minimal comment.